mbta-performance/chalicelib/historic/process.py

```diff
@@ -21,6 +22,8 @@ def process_events(input_csv: str, outdir: str, nozip: bool = False):
     df["event_time"] = df["service_date"] + pd.to_timedelta(df["event_time_sec"], unit="s")
     df.drop(columns=["event_time_sec"], inplace=True)
 
+    df = add_gtfs_headways(df)
```

Comment:
i think we should try to be consistent with how we access GTFS in this repo--ie, either stick to the `mbta_gtfs_sqlite` package or use the gobble code. (im free later this week to work help on this if youd like)

Reply:
This is the code we've been using in t-perforance-dash for the last 2 years, and is what gobble code is based on. We have validated it works well for the bus data, so I think it would be good to stick with this format imo

Reply:
started this in a separate branch! for easier reviewing (: